### PR TITLE
Fix benchmark script build on mac m1

### PR DESCRIPTION
**Pull request by pankajastro**
# Description
closes: https://github.com/astronomer/astro-sdk/issues/834

Let's build the image with `--platform` option since if build on M1 Mac then it is not working in other architecture. For now, I have added the platform value `linux/amd64` if needed we can add multiple values in future like `linux/amd64,linux/arm64 `  

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### CODE REVIEW

File: python-sdk/tests/benchmark/Dockerfile [hunk 1]
```diff
@@ -1,4 +1,4 @@
-FROM debian:bullseye AS build
+FROM --platform=linux/amd64 debian:bullseye AS build
```
Comment (feluelle) [inline python-sdk/tests/benchmark/Dockerfile:1]: You can also pass this option to the `docker` command line which I think is what you want as not everyone uses the same system.

Comment (feluelle) [reply python-sdk/tests/benchmark/Dockerfile:1]: See [docker build](https://docs.docker.com/engine/reference/commandline/build/)

Comment (sunank200) [reply python-sdk/tests/benchmark/Dockerfile:1]: +1

Comment (sunank200) [inline python-sdk/tests/benchmark/Dockerfile:1]: Would be useful to pass this as a docker build option.

**Review by kaxil — APPROVED** [review]

**Comment by pankajkoti** [comment]
Nice fix @pankajastro . Thank you!

File: python-sdk/tests/benchmark/Makefile [hunk 2]
```diff
@@ -5,7 +5,7 @@ GCP_PROJECT ?= astronomer-dag-authoring
 GIT_HASH = $(shell git log -1 --format=%h)
 APP ?= benchmark
 CONTAINER_REGISTRY=gcr.io/$(GCP_PROJECT)/$(APP)
-
+PLATFORM="linux/amd64"
```
Comment (feluelle) [inline python-sdk/tests/benchmark/Makefile:8]: This is not configurable right now, or? Everyone using this script has to run it on m1. Maybe we should make it configurable through an env var.

Comment (pankajastro) [reply python-sdk/tests/benchmark/Makefile:8]: yes, it is not configurable. I think the platform option in the docker build is more about the target machine where we want to run this image?
I think it's ok for now but let me know if you want this value from env I'm happy to do that change and keep `linux/amd64` as the default value

Comment (feluelle) [reply python-sdk/tests/benchmark/Makefile:8]: Yes, I think some of us do not have a m1.

Comment (feluelle) [reply python-sdk/tests/benchmark/Makefile:8]: So it would be nice to be able to set it without editing the script.

Comment (pankajastro) [reply python-sdk/tests/benchmark/Makefile:8]: PR should take care of this https://github.com/astronomer/astro-sdk/pull/1002

**Pull request merged**